### ISSUE TITLE
Custom keybinding for jump to panel instead of 1-5

### ISSUE THREAD

**Issue by pavel-alay** [jesseduffield/lazygit#1455]
I didn't find 'jump to …' among [possible keybindings](https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md#default)
It would be nice to be able to rebind jump to panel to smth else instead of default `1-5`.

Does [Custom Command Keybindings](https://github.com/jesseduffield/lazygit/blob/master/docs/Custom_Command_Keybindings.md) allow to do it?

**Comment by fsmiamoto** [comment]
Looks like currently we don't support it @pavel-alay 

The snippet was taken from the code that  creates the keybindings and we don't use any user config for this case:
```go
for i, window := range []string{"status", "files", "branches", "commits", "stash"} {
	bindings = append(bindings, &Binding{ViewName: "", Key: rune(i+1) + '0', Modifier: gocui.ModNone, Handler: gui.goToSideWindow(window)})
}
```
But I guess it wouldn't it be too difficult to support that.

**Comment by jesseduffield** [comment]
I agree: these should be configurable. Should be simple enough to implement: I'll chuck an Octoberfest label on this

**Comment by mark2185** [comment]
This is solved by #1515 , if I'm not mistaken?

**Comment by jesseduffield** [comment]
@mark2185 indeed, closing
